src/bp/ml/crf-js/addon.ts

```diff
@@ -0,0 +1,34 @@
+const addon = require('./crfsuite.node')
+export default addon as BindingType
+
+type TaggerCtor = new () => Tagger
+type TrainerCtor = new (opt?: TrainerOptions) => Trainer
+
+type BindingType = {
+  Tagger: TaggerCtor
+  Trainer: TrainerCtor
+}
+
+export declare class Tagger {
+  tag(xseq: Array<string[]>): { probability: number; result: string[] }
+  open(model_filename: string): boolean
```

Comment:
that would be amazing if we could load a tagger from a buffer instead of from file. It's ouf of scope of this PR but it would be amazing. It would allow us to avoid creating a tmp file with crf model as buffer before loading

Reply:
Yeah you're right! It sure is doable... I can put this on my TODO list if you want?